### ISSUE TITLE
Can't find rules of tslint-microsoft-contrib when installed as peer.

### ISSUE THREAD

**Issue by IZEDx** [Glavin001/tslint-clean-code#30]
It seems like tslint-clean-code can't find the tslint-microsoft-contrib rules directory, when it's installed as a peer-dependency.

Changing tslint.json line 3 from "node_modules/tslint-microsoft-contrib" to "tslint-microsoft-contrib" fixes that, but not sure if it would then still find it inside of node_modules.

**Comment by Glavin001** [comment]
I have not personally experienced this issue yet.

Pull Requests welcome!

**Comment by Nimelrian** [comment]
Can confirm that tslint-clean-code does not work at all since tslint can't find the contrib rules. As they are only defined as a dev dependency of clean-code, they don't get installed when clean-code is used as a dependency of other projects. Installing the contrib rules myself does not solve the issue (since clean-code looks into its own node_modules directory).

**Comment by Glavin001** [comment]
I checked my other projects using this package and this is how I was using it:

```javascript
{
  "rulesDirectory": [
    "./node_modules/tslint-microsoft-contrib",
    "./node_modules/tslint-clean-code/dist/src"
  ],
  "extends": [
  ],
  "rules": {
    // List all of the specific rules...
  }
}
```

Are you all using something like the following?

```json
{
  "extends": "tslint-clean-code"
}
```

This explains why I have not yet experienced the problem.

**Comment by Nimelrian** [comment]
Yup, I simply have it in my extends list:
```json
"extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier", "tslint-plugin-prettier", "tslint-clean-code"],
```

**Comment by Glavin001** [comment]
I think I am going to adopt `tslint-eslint-rules` way of doing things: https://github.com/buzinas/tslint-eslint-rules/blob/master/index.js

```
// unopinionated extensable tslint configuration
// loads rules for extending packages, but does not enable any
module.exports = {
    rulesDirectory: "./dist/rules",
};
```

Thus, using `extends` as @Nimelrian and others are doing would result into the `rulesDirectory` being added for `tslint-clean-code` however not enabling any of the rules -- which I prefer.

This also removes the requirement for `tslint-microsoft-contrib` as it should have only been a requirement for this repository when contributing and not a requirement for those using `tslint-clean-code` and not wishing to use `tslint-microsoft-contrib`.

Pull Request coming soon!

**Comment by Glavin001** [comment]
Here's my PR: https://github.com/Glavin001/tslint-clean-code/pull/34

Please take a look and 👍 and/or comment! I will wait for some feedback before merging and publishing.

**Comment by Glavin001** [comment]
Published to v0.2.2
